Code/KinectLibraries/KinectDepthImageOutput/CMakeLists.txt.user

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should have been deleted. It is in this folder as Qtcreator the first time run, build in the source folder.